### PR TITLE
fix some build errors

### PR DESCRIPTION
**Pull request by titi4u**
Hi,
when trying to build the Xianxia mod, the build failed because:
- Java memory was too low
- the xml "versionNumber" tag could not be correctly set
- github downloads fail because no secured connection could be set

I thought it would maybe interest you to include my fixes ;)
The only problem that still exists (for me) is that I have to manually set the content of "$sdk" because FLEX_HOME is undefined.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,17 @@ powershell
 Set-ExecutionPolicy unrestricted
 ```
 
+#### If build with PowerShell does not work
```
Comment (Oxdeception) [inline README.md]: This is too similar to a previous heading. Something more along the lines of "Build Errors" would fit better.

File: README.md [hunk 2]
```diff
@@ -46,6 +46,17 @@ powershell
...
+If you get errors like "Error: Java heap space" or "dx tool failed", you probably have to allocate more Java memory:
+- open the jvm.config file inside your Flex Air SDK directory and set at least 512 Mb
```
Comment (Oxdeception) [inline README.md]: This file is not located in the base Flex SDK directory.  It is located in %FLEX_HOME%/bin

Probably should specify to look for the line that says this, for clarity, and specify what exactly this is doing.

File: README.md [hunk 3]
```diff
@@ -46,6 +46,17 @@ powershell
...
+```
+java.args=-Xmx512m
+```
+- create the _JAVA_OPTIONS system environment variable
```
Comment (Oxdeception) [inline README.md]: Is this required? Simply setting the jvm.args should handle this. If this setting is handling the heap space, what is the purpose of the config file and the reverse also?

File: Start.ps1 [hunk 4]
```diff
@@ -73,11 +73,13 @@ function BuildApk
 switch -wildcard (Read-Host "What would you like to do `n1.Download and Build Revamp `n2.Download and Build Xianxia `n3.Build from Source folder `n4.Build apk using CoC-AIR.swf `n5.Clean the Directory`n") 
 { 
     "1*" {
+		[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
```
Comment (Oxdeception) [inline Start.ps1:77]: Please add a comment for the purpose of this line.

File: Start.ps1 [hunk 5]
```diff
@@ -46,7 +46,7 @@ function Setup
 #Builds the Stuff
 function BuildSwf
 {
-	(Get-Content $xml) -replace '<versionNumber>(.*)</versionNumber>', ('<versionNumber>'+((${latestVersion}) -split "_")[-1]+'</versionNumber>')| Set-Content $xml
+	[Regex]::Replace( $(Get-Content $xml), '(?s)/<versionNumber>.*?</versionNumber>/', ('<versionNumber>'+((${latestVersion}) -split "_")[-1]+'</versionNumber>') ) | Set-Content $xml
```
Comment (Oxdeception) [inline Start.ps1:49]: I believe it may be time to split this line with these changes.

**Review by Oxdeception — CHANGES_REQUESTED** [review]
Needs some changes for clarity. 

Also note FLEX_HOME is an environment variable that may need to be manually set if not set by FlashDevelop when installing the SDK.

File: README.md (hunk 1 above)
Comment (titi4u) [reply README.md]: ok I'll do that!

File: README.md (hunk 2 above)
Comment (titi4u) [reply README.md]: Yes, sorry it's in the "bin" directory, I'll correct that.

File: README.md (hunk 3 above)
Comment (titi4u) [reply README.md]: Yes, for me it's needed, I found solution here: http://www.flashdevelop.org/community/viewtopic.php?f=6&t=12995

File: Start.ps1 (hunk 5 above)
Comment (titi4u) [reply Start.ps1:49]: I won't do much changes here, I just wanted to fix the bug the most cleanly possible, that's my first powershell script fix ;)

File: Start.ps1 (hunk 4 above)
Comment (titi4u) [reply Start.ps1:77]: If I don't add this line, I'll get "Invoke-WebRequest : The request was aborted: Could not create SSL/TLS secure channel." At least TLS 1.2 is needed to access to GitHub API endpoints.

**Pull request merged**